jerry-core/jerry.h

```diff
@@ -36,8 +36,7 @@ extern "C"
 #define JERRY_FLAG_SHOW_OPCODES           (1u << 0) /**< dump byte-code to stdout after parse */
 #define JERRY_FLAG_MEM_STATS              (1u << 1) /**< dump memory statistics */
 #define JERRY_FLAG_MEM_STATS_PER_OPCODE   (1u << 2) /**< dump per-instruction memory statistics during execution
-                                                     *   (in the mode full GC is performed after execution of each
-                                                     *   opcode handler) */
+                                                     *   FIXME: Remove. */
```

Comment:
Note that these are bit flags that can be combined (or'd). Do we use enums for such constructs? If we'd like to go for a change then bitfield structs seem a better fit to me. If.